automation/hotswap.sh

```
@@ -5,17 +5,20 @@
 
 set -eu
 
-if [ -z "$1" ]
-    then
-        echo "No arguments supplied. Please provide FIAB name as an argument."
-        exit 1
+# see https://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash
+if [ -z ${1+x} ]; then
```

this was failing silently

---

TIL